packages/server/test/unit/agent_spec.ts

```diff
@@ -0,0 +1,225 @@
+import * as http from 'http'
+import * as https from 'https'
+import * as fs from 'fs'
+import * as os from 'os'
+import * as path from 'path'
+import DebuggingProxy = require('debugging-proxy')
+import { expect } from 'chai'
+import * as express from 'express'
+import * as Promise from 'bluebird'
+import * as request from 'request-promise'
+// @ts-ignore
+import CA = require('../../../https-proxy/lib/ca')
```

Comment:
if you need this thing, then export it from the `main` of the `https-proxy` package